python/doc/examples/data_analysis/smoothing_mixture.ipynb

```diff
@@ -0,0 +1,524 @@
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
```

Comment:
Could you switch the colors and use a dotted curve for the mixture? This would make it consistent with the later graphs.